src/Uno.WinUIRevert/Program.cs

```diff
@@ -0,0 +1,146 @@
+
+			var compositionPath = Path.Combine(basePath, @"src\Uno.UWP\UI\Composition");
+			if (Directory.Exists(compositionPath))
+			{
+				Console.WriteLine(@"Moving composition");
+				Directory.Move(compositionPath, Path.Combine(basePath, @"src\Uno.UI\UI\Composition"));
+			}
+
+			var colorsFilepath = Path.Combine(basePath, @"src\Uno.UWP\UI\Colors.cs");
+			if (File.Exists(colorsFilepath))
+			{
+				File.Copy(colorsFilepath, Path.Combine(basePath, @"src\Uno.UI\UI\Colors.cs"), true);
+			}
+
+			var colorHelperFilePath = Path.Combine(basePath, @"src\Uno.UWP\UI\ColorHelper.cs");
+			if (File.Exists(colorHelperFilePath))
+			{
+				File.Copy(colorHelperFilePath, Path.Combine(basePath, @"src\Uno.UI\UI\ColorHelper.cs"), true);
+			}
+
+			// Generic remplacements
+			var genericReplacements = new[] {
+				("Windows.UI.Xaml", "Microsoft.UI.Xaml"),
+				("Windows.UI.Composition", "Microsoft.UI.Composition"),
+				("Windows.UI.Colors", "Microsoft.UI.Colors"),
+				("Windows.UI.ColorHelper", "Microsoft.UI.ColorHelper"),
+				("Windows.UI.Xaml", "Microsoft.UI.Xaml"),
+				("Windows.UI.Xaml.Controls\", \"ProgressRing", "Uno.UI.Controls.Legacy\", \"ProgressRing"),
+				("<UNO_UWP_BUILD>true</UNO_UWP_BUILD>", "<UNO_UWP_BUILD>false</UNO_UWP_BUILD>"),
+			};
+
+			ReplaceInFolders(basePath, genericReplacements);
+
+			// Restore ProgressRing
+			var progressRingReplacements = new[] {
+				("Microsoft.UI.Xaml.Controls", "Uno.UI.Controls.Legacy"),
+			};
+
+			ReplaceInFolders(Path.Combine(basePath, @"src\Uno.UI\UI\Xaml\Controls\ProgressRing"), progressRingReplacements);
+			ReplaceInFile(Path.Combine(basePath, @"src\Uno.UI\UI\Xaml\Controls\ProgressRing\ProgressRing.xaml"), "\"ProgressRing\"", "\"legacy:ProgressRing\"");
+			ReplaceInFile(Path.Combine(basePath, @"src\Uno.UI\UI\Xaml\Style\Generic\Generic.Native.xaml"), "ProgressRing", "legacy:ProgressRing");
+			ReplaceInFile(Path.Combine(basePath, @"src\Uno.UI\Microsoft\UI\Xaml\Controls\ProgressRing\ProgressRing.xaml"), "using:Windows.UI.Xaml.Controls", "using:Uno.UI.Controls.Legacy");
+
+			// Restore DualPaneView XAML
+			// ReplaceInFile(Path.Combine(basePath, @"src\Uno.UI\Microsoft\UI\Xaml\Controls\TwoPaneView\TwoPaneView.xaml"), "using:Windows.UI.Xaml.Controls", "using:Windows.UI.Xaml.Controls");
+
+			// Adjust Colors
+			ReplaceInFile(Path.Combine(basePath, @"src\Uno.UI\UI\Colors.cs"), "Windows.UI", "Microsoft.UI");
+			ReplaceInFile(Path.Combine(basePath, @"src\Uno.UI\UI\ColorHelper.cs"), "Windows.UI", "Microsoft.UI");
+			ReplaceInFile(Path.Combine(basePath, @"src\SourceGenerators\Uno.UI.SourceGenerators\XamlGenerator\XamlConstants.cs"), "Windows.UI", "Microsoft.UI");
+			ReplaceInFile(Path.Combine(basePath, @"src\Uno.UI\UI\Xaml\Markup\Reader\XamlConstants.cs"), "Windows.UI", "Microsoft.UI");
+
+			// Custom animation
+			// ReplaceInFile(Path.Combine(basePath, @"src\Uno.UI\UI\Xaml\Media\Animation\Animators\RenderingLoopAnimator.wasm.cs"), "Microsoft", "Windows");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5475653)